src/rpc/rawtransaction.cpp

```diff
@@ -561,6 +563,7 @@ static RPCHelpMan decodescript()
             RPCResult::Type::OBJ, "", "",
             {
                 {RPCResult::Type::STR, "asm", "Script public key"},
+                        {RPCResult::Type::STR, "desc", "Inferred descriptor for the script"},
```

Comment:
In c90a08886afe86d84bbd1d76ae3d3ecc24f24a74 "transaction decoding infer output descriptors"

Indentation